std/bitmanip.d

```diff
@@ -1566,6 +1595,7 @@ public:
         return result;
     }
 
+    ///
     @system unittest
```

Comment:
What on Earth is `opCom` and why is it associated with `~`?

Reply:
Seems to be a real thing: https://github.com/dlang/dmd/blob/25bc0213a7455b8ccbb9a4cc869c86ecc24c6777/src/dmd/id.d#L205

(and it is called from the tests)

Reply:
Looks like DMD supports explicitly overloading ~ with `opCom`:

https://github.com/dlang/dmd/blob/25bc0213a7455b8ccbb9a4cc869c86ecc24c6777/src/dmd/expression.d#L5756

Reply:
See also: https://issues.dlang.org/show_bug.cgi?id=18665